fvm/src/executor/default.rs

```diff
@@ -186,21 +180,19 @@ where
     }
 
     // TODO: The return type here is very strange because we have three cases:
-    // 1. Continue (return actor ID & gas).
-    // 2. Short-circuit (return ApplyRet).
-    // 3. Fail (return an error).
-    //
-    // We could use custom types, but that would be even more annoying.
+    //  1. Continue (return actor ID & gas).
+    //  2. Short-circuit (return ApplyRet).
+    //  3. Fail (return an error).
+    //  We could use custom types, but that would be even more annoying.
```

Comment:
I generally prefer adding line breaks between sections. Otherwise, it's hard to tell sections/paragraphs apart.